hazelcast/src/test/java/com/hazelcast/config/GlobalSerializerConfigTest.java

```diff
@@ -16,13 +16,19 @@
 
 package com.hazelcast.config;
 
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
 import nl.jqno.equalsverifier.EqualsVerifier;
 import nl.jqno.equalsverifier.Warning;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
 
-import static com.hazelcast.test.HazelcastTestSupport.assumeDifferentHashCodes;
-
-public class GlobalSerializerConfigTest {
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class GlobalSerializerConfigTest extends HazelcastTestSupport {
```

Comment:
We should not add the parent class here when only a static method is used. The `HazelcastTestSupport` is a really fat class.